documentation/src/main/asciidoc/topics/con_cache_encoding.adoc

```diff
@@ -0,0 +1,41 @@
+[id='encoding']
+= Cache encoding
+Encoding is the format, or media type, that {brandname} uses to store entries (key/value pairs) in caches.
+
+[discrete]
+== Remote caches
+
+{brandname} Server stores entries in remote caches without any media type information by default.
+
+Encoding configures how {brandname} stores entries in a remote cache.
+If you do not configure encoding for remote caches then {brandname} Server stores entries however clients provide them.
+In this case {brandname} Server also returns entries as they are stored in the remote cache for any client requests.
+With encoding, {brandname} converts keys and values, when necessary, to whatever media type you specify for the cache.
+
+[TIP]
+====
+Always configure an encoding for remote caches if you want to use multiple clients, such as {brandname} Console or CLI, Hot Rod, REST, as well as server-side tasks and scripts.
+
+{brandname} recommends application/x-protostream`.
+====
+
+[discrete]
+== Embedded caches
+
+{brandname} stores entries in embedded caches as Plain Old Java Objects (POJOs) by default.
+
+Storing POJOs in embedded caches means that object instances are mutable.
+For example if you store a `HashMap` any modifications to the map are seen by other users of the cache, causing race conditions.
+
+However, for clustered embedded caches, {brandname} needs to serialize any POJOs to a byte array that can be replicated between nodes and then deserialized back into POJOs.
+This requires you to configure a marshaller to perform serialization and deserialization for your embedded caches.
```

Comment:
A marshaller is configured by default .. I think it's more appropriate to say that you need to ensure that your POJOs are serializable via one of our supported marshallers.

Reply:
How about "This means you must ensure that {brandname} can use a marshaller to serialize and deserialize your POJOs." ?

Not sure if we should say "supported" in community docs. It's also a problematic phrase downstream.

Reply:
"This means that you must ensure that {brandname} is able to serialize your POJOs using the configured marshaller."

You could then re-iterate that if no marshaller is configured, the default is Protostream?

Reply:
Maybe a more concise wording is "This means you must ensure that {brandname} can serialize your POJOs with the ProtoStream marshaller if you do not configure another marshaller."

Reply:
@ryanemerson I think the updated changes should do it. would you mind double-checking? thanks.

Reply:
Looks good :+1: